lib/config/index.js

```diff
@@ -136,4 +138,30 @@ function checkLimitsConfig (cfg) {
   }
 }
 
+/** Check the format of the error-codes config files
+ *
+ */
+
+function checkErrorCodes() {
+
+  for (const domain of Object.keys(errorCodes)) {
+    let name = errorCodes[domain];
+    if (!name.hasOwnProperty('subdomains') || !name.hasOwnProperty('code')) {
+      throw new KuzzleInternalError(`Invalid configuration: Missing mandatory field in ${domain} error-codes config file.`, name, 0);
```

Comment:
```suggestion
      throw new KuzzleInternalError(`Invalid configuration: Missing required field in ${domain} error-codes config file.`, name, 0);
```

Reply:
you cannot use an invalid error name, instead you need to fall back on an error name+code dedicated to describe an invalid error file

(same thing for other errors thrown below)

Reply:
I'm not sure I get what you meant

Reply:
The error name you provide does not exist, and the "0" error code doesn't seem right either.

You need to create a dedicated error name and error code for that kind of errors.

Reply:
for now i change it to 
`throw new KuzzleInternalError(`Field ${domain} wrongly formatted.`, 'Invalid error file', 1)`
Since my check function is aside the error files loader, I can't call the `throw` method of `kuzzle`, I can't see another way to do it properly

Reply:
I think it's more explicit to do it this way, crashing with an `InternalError`, and yes for now I can't see another case we would need to do it